examples/community/README.md

```diff
@@ -3279,7 +3281,7 @@ export_to_video(
 
 ### StyleAligned Pipeline
 
-This pipeline is the implementation of [Style Aligned Image Generation via Shared Attention](https://arxiv.org/abs/2312.02133).
+This pipeline is the implementation of [Style Aligned Image Generation via Shared Attention](https://arxiv.org/abs/2312.02133). Checkout [this](https://github.com/huggingface/diffusers/pull/6489) PR for more examples and results.
```

Comment:
Was this just added?

Reply:
Yes. I thought it'd be okay since I forgot doing it on the StyleAligned PR. Should I remove it?

Reply:
Yes please. Then we can merge 👍🏽

Reply:
hi @DN6, done. lgtm!